src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+
+        ball.setColor(ColorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100));
```

Comment:
Magic numbers should be avoided. You should declare a constant for the number 100 with a clear name that indicates its purpose.